### ISSUE TITLE
Unable to replicate the rusult in table 2

### ISSUE THREAD

**Issue by MingZeGao111** [GeJulia/flc_pooling#6]
Sorry to bother yor again. Now I only use test dataset to test the accuracy of Preact-resnet-18 with FLC pooling on CIFAR-10, I got 83.9% clean accuracy(similar to that in your paper), but I got 1.88%PGD accuracy which is different from the result in your paper. The parameter is mentioned before, what's the problem?

**Comment by GeJulia** [comment]
Did you make sure to train only with the training data and not like in testing use both data streams?

**Comment by MingZeGao111** [comment]
Yes! But I still get the above rusult. 
![3a63cef94b8e0a1f295778bb1892903](https://github.com/GeJulia/flc_pooling/assets/163491915/b2731aa6-a8a8-4503-9230-1dc064b579d8)
![f4637ebef789e1ccb288f2902664eea](https://github.com/GeJulia/flc_pooling/assets/163491915/c3a0594b-a5de-4700-92cd-b4c260757902)

![a52bcece5843d45fb24fc672ad37388](https://github.com/GeJulia/flc_pooling/assets/163491915/3799b4e4-5305-4769-9386-b5bb7e1d52b5)

**Comment by GeJulia** [comment]
The training script looks fine. I'm not sure what is the issue here. One suggestion would be removing the <norm='forward'> in the FFT and IFFT operation and training the network again.

**Comment by MingZeGao111** [comment]
what's the difference between <norm='forward'> and <norm='backward'>(Default)

**Comment by GeJulia** [comment]
The normalization in the FFT and IFFT transform, as stated:

https://pytorch.org/docs/stable/generated/torch.fft.fft2.html
https://pytorch.org/docs/stable/generated/torch.fft.ifft2.html
